continuous_integration/environment-3.9-dev.yaml

```diff
@@ -5,9 +5,9 @@ channels:
 dependencies:
 - c-compiler
 - dask>=2022.3.0
-# FIXME: handling is needed for httpx-based fastapi>=0.87.0
-- fastapi>=0.69.0,<0.87.0
+- fastapi>=0.92.0
```

Comment:
Noticing that allowing any `fastapi` version `0.92.0` or greater results in [a bunch of test failures]( https://github.com/dask-contrib/dask-sql/actions/runs/5685404699/job/15410229312?pr=1205#step:8:42129 )

Reply:
IIUC, those test failures are unrelated and caused by (what I assume) is new default handling for pyarrow strings in Dask, shouldn't be too difficult to resolve and then merge into this PR

Reply:
Opened https://github.com/dask-contrib/dask-sql/pull/1206 to verify that these issues are isolated from the `fastapi` version; if things are green there, can merge with this branch and push on the one remaining test failure

Reply:
Fixed by PR ( https://github.com/dask-contrib/dask-sql/pull/1206 ). Marking resolved